.github/workflows/build.yml

```diff
@@ -53,9 +53,12 @@ jobs:
         with:
           name: project
 
+      - name: Install gotestsum
+        run: go install gotest.tools/gotestsum@latest
+
       - name: Run tests
         continue-on-error: ${{ github.event_name != 'pull_request' && 'true' || 'false' }}
-        run: go test -covermode atomic -coverprofile=profile.cov ./pkg/...
+        run: ~/go/bin/gotestsum -- -covermode atomic -coverprofile=profile.cov ./pkg/...
```

Comment:
can we generate an XML output while also keep outputting test results to the console log?

using something like this `~/go/bin/gotestsum --junitfile test.xml`

Reply:
yes